config/example.yaml

```diff
@@ -21,6 +21,7 @@ grains:
   absolute_smallest_grain_size: 100
   background: 0.0
   zrange: [0, 3] # List of low and high height range
+  cropped_size: 40
```

Comment:
We should perhaps indicate whether this (and `absolute_smallest_grain_size`) is in pixels or nanometres as a comment after the value.

Reply:
Yes very true. I was going to wait to add it into Bob's PR surrounding info on config file inputs but it being here would also certainly be useful

Reply:
Added `# Length (in nm) of square cropped images (can take -1 for grain-sized box)` in recent commit